docs/core/pr-review.md

```diff
@@ -6,7 +6,9 @@ When someone [opens a pull request](/tools/git.md#pr) (aka "PR") on CiviCRM Core
 * A [CiviCRM Development Environment](https://github.com/civicrm/civicrm-buildkit/blob/master/doc/civibuild.md) (this might be optional, but good to have). One benefit is the ability to check out the PR in your environment.
 * Comfort reading code and patches
 
-Follow the remaining the steps below to review a pull request.
+## Video walk-through
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/s7Tt5PGfHl4?rel=0" frameborder="0" allow="encrypted-media" allowfullscreen></iframe>
```

Comment:
@seanmadsen would it be better having `width="100%"` and no height? Not sure what sort of css we have on just trying to think responsively